### PR TITLE
#35 Created footer component

### PR DESCRIPTION
**Pull request by djek195**
1. Created web and mobile version
2. This component without logo yet, but it has container

### CODE REVIEW

**Review by vita792 — APPROVED** [review]

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        <div className={styles.logoAndTime}>
+            <img src="" alt="MEGASTORE" />
+            <p>Пн-Нд 09:00-20:00</p>
+        </div>
+        <div className={styles.info}>
+            <ul>Інформація про компанію
+                <li><a href="#">Про нас</a></li>
+                <li><a href="#">Умови використання сайту</a> </li>
+                <li><Icon className={styles.icon} icon={"fluent:call-12-filled"}/> <a href="tel:+380939475397">+380 939 475 397</a></li>
+                <li><Icon className={styles.icon} icon={"uiw:mail"}/> <a href="mailto:megamarket@market.com">megamarket@market.com</a></li>
```
Comment (halyna-k) [inline src/components/Footer/Footer.js:16]: to make the code look cleaner, it is better to write List using .map

File: src/components/Footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            </ul>
+        </div>
+        <div className={styles.deliveryAndPayment}>
+            <ul> Доставка та оплата
+                <li><a href="#">Гарантія</a></li>
+                <li><a href="#">Повернення товару</a></li>
```
Comment (halyna-k) [inline src/components/Footer/Footer.js:22]: same here

**Review by halyna-k — APPROVED** [review]

File: src/components/Footer/Footer.js (hunk 1 above)
Comment (djek195) [reply src/components/Footer/Footer.js]: done

File: src/components/Footer/Footer.js (hunk 2 above)
Comment (djek195) [reply src/components/Footer/Footer.js]: done

**Review by halyna-k — APPROVED** [review]

**Pull request merged**